src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -2825,6 +2825,24 @@ We use App Center to track which settings are being used, find bugs, and fix cra
   <data name="BehaviorForOpeningFilesAndFolders" xml:space="preserve">
     <value>Behavior for opening files and folders</value>
   </data>
+  <data name="AddSingleItemToArchive.Text" xml:space="preserve">
```

Comment:
```suggestion
  <data name="AddSingleItemToArchive" xml:space="preserve">
```

Reply:
Should I change also `AddToArchive.Text` to `AddToArchive`?

Reply:
Yes 👍
Btw, is it posible to leave the conversations as unresloved so it's easier to find them later? Thank you 🙂